### DISCUSSION TITLE
Having trouble using the https://sloc.xyz/ api with gitlab

### DISCUSSION THREAD

**Discussion by VixieTSQ** [boyter/scc#312] [Q&A] [answered]
I'm using `https://sloc.xyz/gitlab/VixieTSQ/acabot/?category=code` which should work but I'm guessing it just isn't very stable with gitlab. Am I doing something wrong or is this an internal problem?

**Comment by boyter** [comment, marked as answer]
Honestly its probably due to how I implemented it. Its a Python lambda function running on AWS which calls out to scc after using git to clone things.

It can be slow the first time it runs, and then work the second, which may be an issue depending on how the badge is saved. It looks like its working for me now though?

Its one of those things I really need to fix with a real solution, but ideally one thats cheap.

**Reply by VixieTSQ** [reply]
it is now magically working for me. Great.
But it'd also be great if you'd implement like an ignore parameter. so I can ignore documentation files and what not. (or rather really a path parameter pointing towards where it SHOULD look)
It seems like the lines it gives me is way above what is actually in my repository right now.

**Reply by boyter** [reply]
I suspect you ran into a cold start issue on the lambda, or it ran in the background and finished things off and cached the result.
